src/py42/clients/users.py

```diff
@@ -244,3 +244,57 @@ def change_org_assignment(self, user_id, org_id):
         uri = u"/api/UserMoveProcess"
         data = {u"userId": user_id, u"parentOrgId": org_id}
         return self._session.post(uri, data=json.dumps(data))
+
+    def get_available_roles(self):
+        """Report the list of roles that are available for the authenticated user to
+        assign to other users.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#UserMoveProcess-post>`__
+
+        Returns:
+            :class:`py42.response.Py42Response`
+        """
+        uri = u"/api/v4/role/view"
+        return self._session.get(uri)
+
+    def get_roles(self, user_id):
+        """Return the list of roles that are currently assigned to the given user.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#UserRole-get>`__
+
+        Args:
+            user_id (int): An ID for a user.
```

Comment:
I think at some point we should make these all be `int | str` since I think it works with both and this way you don't have to do unnecessary transformations if they arise